src/libs/OptionsListUtils.ts

```diff
@@ -96,81 +154,80 @@ Onyx.connect({
     },
 });
 
-let allTransactions = {};
+let allTransactions: OnyxCollection<Transaction> = {};
 Onyx.connect({
     key: ONYXKEYS.COLLECTION.TRANSACTION,
     waitForCollectionCallback: true,
-    callback: (val) => {
-        if (!val) {
+    callback: (value) => {
```

Comment:
Consider renaming `value` to `transactions`, as it is easier to understand

Reply:
this is a pattern which is used in whole application ,  I would leave it to be more consistent , wdyt?